tests/user/test_user.yml

```diff
@@ -84,6 +89,7 @@
   - name: User pinky present with changed settings
     ipauser:
       ipaadmin_password: SomeADMINpassword
+      ipaapi_context: "{{ ipa_context | default(omit) }}"
       name: pinky
       first: pinky
       last: Acme
```

Comment:
We should change the SSH keys. `pinky@ipaserver.el81.local` might not be optimal.

Reply:
I agree. Although these are not used, other than storing the keys.